examples/api-arg-many/index.rsh

```diff
@@ -0,0 +1,46 @@
+'reach 0.1'
+
+export const main = Reach.App(() => {
+  const Deployer = Participant('Deployer', {
+    ready: Fun([], Null),
+  })
+  const A = API('A', {
+    sumMany: Fun([UInt, UInt, UInt, UInt,
+                  UInt, UInt, UInt, UInt,
+                  UInt, UInt, UInt, UInt,
+                  UInt, UInt, UInt, UInt,
+                  // I wanted to test more than 15, but when 16 worked I
+                  // decided to keep going higher.
+                  UInt, UInt, UInt, UInt,
+                 ],
+                 UInt)
+  })
+  setOptions({
+    connectors: [
+      // With 16 arguments, the Solidity compiler errors saying that the stack is too deep.
```

Comment:
You should fix that

Reply:
I've been looking at this. I don't know if we can get around this in Solidity.  A relevant issue I came across is [1], which says that it's a problem with accessing more than 16 things on the stack that could be solved with this draft EIP [2].  That tells me that we can use workarounds discussed in those pages if we generate EVM code directly, but probably not with Solidity.  That said, I don't know that much about Solidity.  Can we play a trick in Solidity like turning one item on the stack into a tuple or something?  If so then maybe we can just do that.

[1] https://github.com/ethereum/solidity/issues/7624
[2] https://eips.ethereum.org/EIPS/eip-663

Reply:
We already do stuff like that. That's what memvars and the depthOf type class are about. We just need to use them more effectively for arguments.

Reply:
(It may be that the way to do this is for Reach/ETH to do the same thing as Reach/ALGO where if you make more arguments we tuple the last one, but I don't know what the rest of the ETH world does)